public/JobHandler/callbacks/open.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from "react";
+import { CallbackType, TaskResult } from "../interface";
+import { OpenJobMetaData } from "../../models/interfaces";
+import { CommonService } from "../../services";
+import { triggerEvent, EVENT_MAP } from "../utils";
+import { DetailLink } from "../components/DetailLink";
+import { ErrorToastContentForJob } from "../components/ErrorToastContentForJob";
+
+type OpenTaskResult = TaskResult<{
+  acknowledged: boolean;
+  shards_acknowledged: boolean;
+}>;
+
+export const callbackForOpen: CallbackType = async (job: OpenJobMetaData, { core }) => {
+  const extras = job.extras;
+  const commonService = new CommonService(core.http);
+
+  const tasksResult = await commonService.apiCaller<OpenTaskResult>({
+    endpoint: "transport.request",
+    data: {
+      path: `/.tasks/_doc/${extras.taskId}`,
```

Comment:
query system index is not recommended. use `GET /_tasks/<task_id>` instead

Reply:
There is some difference between /.tasks/_doc/${extras.taskId} and /_tasks/<task_id>, will make it a task in 2.9, let's keep it in 2.8

Reply:
The difference is pretty minimal:

```
➜  config git:(main) ✗ curl --cert ./kirk.pem --key ./kirk-key.pem -XGET https://localhost:9200/.tasks/_doc/0nozY80cTViD5VPloPwRYA:6475 -k
{"_index":".tasks","_id":"0nozY80cTViD5VPloPwRYA:6475","_version":1,"_seq_no":0,"_primary_term":1,"found":true,"_source":{"completed":true,"task":{"node":"0nozY80cTViD5VPloPwRYA","id":6475,"type":"transport","action":"indices:admin/open","description":"open indices [sample-index]","start_time_in_millis":1730387678257,"running_time_in_nanos":203230625,"cancellable":false,"cancelled":false,"headers":{}},"response":{"acknowledged":true,"shards_acknowledged":true}}}%
➜  config git:(main) ✗ curl --cert ./kirk.pem --key ./kirk-key.pem -XGET https://localhost:9200/_tasks/0nozY80cTViD5VPloPwRYA:6475 -k
{"completed":true,"task":{"node":"0nozY80cTViD5VPloPwRYA","id":6475,"type":"transport","action":"indices:admin/open","description":"open indices [sample-index]","start_time_in_millis":1730387678257,"running_time_in_nanos":203230625,"cancellable":false,"cancelled":false,"headers":{}},"response":{"acknowledged":true,"shards_acknowledged":true}}
```